src/Checkout/PayPalOrderTransactionCaptureService.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Swag\PayPal\Checkout;
+
+use Shopware\Core\Checkout\Order\Aggregate\OrderTransactionCapture\OrderTransactionCaptureService;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepositoryInterface;
+use Swag\PayPal\SwagPayPal;
+
+class PayPalOrderTransactionCaptureService
+{
+    /**
+     * @var EntityRepositoryInterface
+     */
+    private $orderTransactionCaptureRepository;
+
+    /**
+     * @var OrderTransactionCaptureService
+     */
+    private $orderTransactionCaptureService;
+
+    public function __construct(
+        EntityRepositoryInterface $orderTransactionCaptureRepository,
+        OrderTransactionCaptureService $orderTransactionCaptureService
+    ) {
+        $this->orderTransactionCaptureRepository = $orderTransactionCaptureRepository;
+        $this->orderTransactionCaptureService = $orderTransactionCaptureService;
+    }
+
+    public function createOrderTransactionCaptureForFullAmount(string $orderTransactionId, Context $context): string
```

Comment:
why adding these first three methods? they do nothing additional to the core service

Reply:
Indeed, i changed the calling code to use the OrderTransactionCaptureService directly 👍